Library/ViewModels/PledgePaymentPlansOptionViewModel.swift

```diff
@@ -80,9 +137,45 @@ private func getTitleText(by type: PledgePaymentPlansType) -> String {
 }
 
 // TODO: add strings translations [MBL-1860](https://kickstarter.atlassian.net/browse/MBL-1860)
-private func getSubtitleText(by type: PledgePaymentPlansType) -> String {
+private func getSubtitleText(by type: PledgePaymentPlansType, isSelected: Bool) -> String {
   switch type {
   case .pledgeInFull: ""
-  case .pledgeOverTime: "You will be charged for your pledge over four payments, at no extra cost."
+  case .pledgeOverTime: {
+      let subtitle = "You will be charged for your pledge over four payments, at no extra cost."
+      guard isSelected else { return subtitle }
+
+      return "\(subtitle)\n\nThe first charge will be 24 hours after the project ends successfully, then every 2 weeks until fully paid. When this option is selected no further edits can be made to your pledge."
+    }()
+  }
+}
+
+private func formattedPledgePaymentIncrement(
+  _ increment: PledgePaymentIncrement,
+  at index: Int,
+  project: Project
+) -> PledgePaymentIncrementFormatted {
+  PledgePaymentIncrementFormatted(from: increment, index: index, project: project)
+}
+
+private func getDateFormatted(_ timeStamp: TimeInterval) -> String {
+  Format.date(
+    secondsInUTC: timeStamp,
+    dateStyle: .medium,
+    timeStyle: .none
+  )
+}
+
+extension PledgePaymentIncrementFormatted {
```

Comment:
This looks like it could live in its own file (and probably have a test or two)

Reply:
Good point! I’ll suggest moving this into its own file and adding a tests as part of a tech-debt task, if that sounds good to you. Let me know if you'd prefer we tackle it sooner.